munininfluxdb/munin.py

```diff
@@ -1,77 +1,125 @@
 from __future__ import print_function
+from collections import namedtuple
 import os
 import sys
 import pprint
 
 from utils import ProgressBar, Symbol
 from settings import Settings
 
-from vendor import storable
+import storable
```

Comment:
Again, `storable` is now available in pypi and has been added to the setup.py dependencies.